src/main/java/uk/gov/hmcts/reform/notificationservice/model/servicebus/in/ErrorCode.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.notificationservice.model.servicebus.in;
```

Comment:
perhaps

```suggestion
package uk.gov.hmcts.reform.notificationservice.model.common;
```

? can be used for outgoing request too. what do you think?

Reply:
may be we put enum and some other data in common but this message  is specific .
for message
uk.gov.hmcts.reform.notificationservice.model.request.incoming might be better?
uk.gov.hmcts.reform.notificationservice.model.common for Errorcode ?

Reply:
or we can review later ;). can always refactor